server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -344,12 +344,18 @@ public void run() {
                 TServerConnection client =
                     manager.tserverSet.getConnection(location.getServerInstance());
                 if (client != null) {
-                  Manager.log.trace("[{}] Requesting TabletServer {} unload {} {}", store.name(),
-                      location.getServerInstance(), tls.extent, goal.howUnload());
-                  client.unloadTablet(manager.managerLock, tls.extent, goal.howUnload(),
-                      manager.getSteadyTime());
-                  unloaded++;
-                  totalUnloaded++;
+                  try {
+                    Manager.log.trace("[{}] Requesting TabletServer {} unload {} {}", store.name(),
+                        location.getServerInstance(), tls.extent, goal.howUnload());
+                    client.unloadTablet(manager.managerLock, tls.extent, goal.howUnload(),
+                        manager.getSteadyTime());
+                    unloaded++;
+                    totalUnloaded++;
+                  } catch (TException tException) {
+                    Manager.log.warn("[{}] Failed to request tablet unload {} {} {} {}",
```

Comment:
If this is going to be in a loop and / or retried? Think that logging at a lower level may be appropriate. Particularity if the RPC could be transient.

Reply:
yes, the tablet group watcher constantly scans the metadata table looking for things to do per tablet.  This throwing an exception for one tablet prevents progress on other tablets.

Reply:
Thinking of lowering the log message from warn to debug or info since RPC problems are expected.